packages/react-search-ui/react-search-ui.d.ts

```diff
@@ -0,0 +1,345 @@
+declare module "@elastic/react-search-ui" {
+
+  import { Component } from 'react';
+
+  type func = (...args: any[]) => any;
+  type renderFunc = (...args: any[]) => JSX.Element;
+
+  /**
+   * FIELDS
+   */
+  export type FieldValue = string | number | boolean | Array<string | number | boolean>;
+
+  export interface FieldValueWrapper {
+    // A raw field value, like 'I am a raw result', or 2, or true. Raw values may
+    // or may not be html escaped, so *always* sanitize a raw value before rendering
+    // it on a page as html.
+    raw?: FieldValue;
+    // A snippet value contains a highlighted value. I.e., 'I <em>am</em> a raw
+    // result'. These are always sanitized and safe to render as html.
+    snippet?: string;
+  }
+
+  /**
+   * FACETS
+   */
+  export type FacetType = "range" | "value";
+
+  export interface FacetValue {
+    // Number of results for this filter
+    count: number;
+    // Filter to apply if selected
+    value: FilterValue;
+    // Whether or not this facet value is selected
+    selected?: boolean;
+  }
+
+  interface FacetT {
```

Comment:
In my own code, I found that I actually needed this type for something - think we should export it like the rest? (generally, the rule is to export everything, I just didn't originally because the source code's class is named `Facet`)